packages/commerce-sdk-react/src/auth/storage/cookie.ts

```diff
@@ -35,6 +35,16 @@ export class CookieStorage extends BaseStorage {
     }
     delete(key: string) {
         const suffixedKey = this.getSuffixedKey(key)
-        Cookies.remove(suffixedKey)
+
+        Cookies.remove(suffixedKey, {
+          ...getDefaultCookieAttributes(),
+          ...options
```

Comment:
It looks like you are using the options object from the constructor here. Did you mean to do that? Or did you want to add an additional parameter like the set function?

Reply:
thanks, I added the additional parameter in the next commit